core/unit_test/TestGraph.hpp

```diff
@@ -214,9 +214,14 @@ TEST_F(TEST_CATEGORY_FIXTURE(count_bugs), zero_work_reduce) {
 // UVM works on pre pascal cards.
 #if defined(KOKKOS_ENABLE_CUDA) && defined(KOKKOS_ENABLE_CUDA_UVM) && \
     (defined(KOKKOS_ARCH_KEPLER) || defined(KOKKOS_ARCH_MAXWELL))
-  Kokkos::fence();
+  if constexpr (std::is_same_v<TEST_EXECSPACE, Kokkos::Cuda>) Kokkos::fence();
+#endif
+// FIXME_HPX graph.submit() isn't properly enqueued
```

Comment:
Nitpick: I would have put the comment on the same line as the `#ifdef` (if that fits without breaking the line)